sidebars.ts

```diff
@@ -293,20 +293,17 @@ const sidebars: SidebarsConfig = {
                 'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-an-agent-with-artifactory-access',
                 'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configuring-artifactory-with-recipes',
                 'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-an-agent-with-pypi-access-visualizations',
-                'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-agent-files-service',
-                'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-organizations-service',
+                'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-organizations-hierarchy',
                 'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-an-agent-with-generic-http-tools-for-use-in-recipes',
                 'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-an-agent-with-strict-recipe-sources',
                 'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-an-agent-with-client-ssl-certificates',
                 'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-an-agent-with-a-moderne-tenant-ssl-certificate',
                 'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-an-agent-to-connect-to-moderne-via-an-http-proxy',
                 'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/agent-variables',
                 'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/multi-tenant-private-recipes',
+                'administrator-documentation/moderne-platform/how-to-guides/agent-configuration/org-service-migration',
               ],
             },
-            'administrator-documentation/moderne-platform/how-to-guides/organizations-service',
```

Comment:
We are removing a link to this in the sidebar without actually removing the file. Intentional? Is the idea that we want to keep it around for DX?